fdbclient/vexillographer/fdb.options

```diff
@@ -223,6 +223,8 @@ description is not currently required but encouraged.
             description="Enables tracing for this transaction and logs results to the client trace logs. The DEBUG_TRANSACTION_IDENTIFIER option must be set before using this option, and client trace logging must be enabled to get log output." />
     <Option name="transaction_logging_max_field_length" code="405" paramType="Int" paramDescription="Maximum length of escaped key and value fields."
             description="Sets the maximum escaped length of key and value fields to be logged to the trace file via the LOG_TRANSACTION option, after which the field will be truncated. A negative value disables truncation." />
+    <Option name="set_as_debug_transaction" code="406"
```

Comment:
I don't mind this name on its own, but I worry that we have some similar features in this vein using similar names that might get confusing. In particular for this one, we have `debug_transaction_identifier` which adds an ID to the transaction that is different than the ID that gets used here, even though by name it seems like there would be some relation between them.

Maybe if we called it something like "server_request_tracing" to 1) make it sound separate from the other feature and 2) make it clear that the purpose of this is to track this request through the processes in the cluster and not for client-side debugging.

Note that when functions are generated to enable these options, they will prefix the function with the work `set`. As a result, you wouldn't want to put that in the name here.

Reply:
Yeah, I got confused by `debug_transaction_identifier` already (see issue). `server_request_tracing` sounds go to me if no one has any objections.

Good to know re `set`. The API I'm using passes in an enum into a generic `set_option()` API.